x-pack/plugins/security_solution/public/timelines/components/timeline/data_providers/providers.tsx

```diff
@@ -115,150 +138,182 @@ TimelineEuiFormHelpText.displayName = 'TimelineEuiFormHelpText';
-                                      : group[0].queryMatch.operator ?? IS_OPERATOR
-                                  }
-                                  register={dataProvider}
-                                  providerId={index > 0 ? group[0].id : dataProvider.id}
-                                  timelineId={id}
-                                  toggleEnabledProvider={() =>
-                                    index > 0
-                                      ? onToggleDataProviderEnabled({
-                                          providerId: group[0].id,
-                                          enabled: !dataProvider.enabled,
-                                          andProviderId: dataProvider.id,
-                                        })
-                                      : onToggleDataProviderEnabled({
-                                          providerId: dataProvider.id,
-                                          enabled: !dataProvider.enabled,
-                                        })
-                                  }
-                                  toggleExcludedProvider={() =>
-                                    index > 0
-                                      ? onToggleDataProviderExcluded({
-                                          providerId: group[0].id,
-                                          excluded: !dataProvider.excluded,
-                                          andProviderId: dataProvider.id,
-                                        })
-                                      : onToggleDataProviderExcluded({
-                                          providerId: dataProvider.id,
-                                          excluded: !dataProvider.excluded,
-                                        })
-                                  }
-                                  val={
-                                    dataProvider.queryMatch.displayValue ??
-                                    dataProvider.queryMatch.value
-                                  }
-                                />
-                              </EuiFlexItem>
-                              <EuiFlexItem grow={false}>
-                                {!snapshot.isDragging &&
-                                  (index < group.length - 1 ? (
-                                    <AndOrBadge type="and" />
-                                  ) : (
-                                    <LastAndOrBadgeInGroup>
-                                      <AndOrBadge type="and" />
-                                    </LastAndOrBadgeInGroup>
-                                  ))}
-                              </EuiFlexItem>
-                            </EuiFlexGroup>
-                          </div>
-                        )}
-                      </Draggable>
-                    ))}
-                    {droppableProvided.placeholder}
-                  </DroppableContainer>
+          <>
```

Comment:
<img width="855" alt="Screenshot 2020-07-01 at 10 13 06" src="https://user-images.githubusercontent.com/6295984/86227040-6e5eb880-bb84-11ea-9288-e24753ca7b53.png">

Reply:
@angorayc is right, you can use React.Fragment and you can add a key here.